monsieurkayode/AritGeo-Kayode-Adeola
# Fix bug in logic and comment

#### What does this PR do?
Fix bug in logic
#### Description of Task to be completed?
Modify function in main module to cater for edge cases
#### How should this be manually tested?
Clone Repo
Cd to directory
Run `npm install`
Run `npm test`
All test should be passing
